app/components/provider_interface/application_card_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="app-application-card">
+
+  <h3 class="govuk-heading-m">
+    <%= link_to candidate_name, provider_interface_application_choice_path(application_choice), class: 'govuk-link' %>
```

Comment:
We have a `ViewHelper` for adding the `govuk-link` class.

```suggestion
    <%= govuk_link_to candidate_name, provider_interface_application_choice_path(application_choice) %>
```

Reply:
Thanks :) will change this.

Reply:
👍